### ISSUE TITLE
🐛 bug report: [HMR] Syntax error in IE11

### ISSUE THREAD

**Issue by maximilianschmitt** [parcel-bundler/parcel#348]
IE11 shows a syntax error. It goes away when using `--no-hmr`.

<img width="949" alt="screen shot 2017-12-19 at 18 27 08" src="https://user-images.githubusercontent.com/3694962/34170222-54d9ba04-e4ea-11e7-8388-6d03cc23f07d.png">

### 🎛 Configuration (.babelrc, package.json, cli command)

No .babelrc

**index.html**

```
<html>
<body>
  <script src="./index.js"></script>
</body>
</html>
```

**index.js**

```
console.log("hello")
```

### 🔦 Context

To test my app in IE11 I have to turn off HMR.

### 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 1.2.1
| Node             | 8.9.3
| Yarn         | 1.2.1
| Operating System | macOS

**Comment by shawwn** [comment]
Could you paste the full compiled file in a gist? Or the lines around where the error is happening? I don't have a windows box handy to test with.

**Comment by shawwn** [comment]
(Or rather, please paste disc/e114665f*.js, since it's not clear from the screenshot what's triggering the problem.)

**Comment by phocks** [comment]
I got the same error. I believe it has to do with this arrow function in the hot reload code that IE11 doesn't know how to process.

```
if (data.type === 'reload') {
      ws.close();
      ws.onclose = () => {
        window.location.reload();
      }
    }
```

**Comment by phocks** [comment]
Seem like this code has already been fixed in the latest master https://github.com/parcel-bundler/parcel/blob/9f5b334d6e04136d93ae428a5eef899371b400e0/src/builtins/hmr-runtime.js#L34-L39

```
if (data.type === 'reload') {
      ws.close();
      ws.onclose = function () {
        window.location.reload();
      }
}
```

**Comment by brandon93s** [comment]
Fixed by #343.
